pex/pep_376.py

```diff
@@ -15,19 +15,12 @@
 from fileinput import FileInput
 
 from pex import dist_metadata, hashing
-from pex.common import (
-    filter_pyc_dirs,
-    filter_pyc_files,
-    find_site_packages,
```

Comment:
Can we delete `find_site_packages` ?

Reply:
Nope, vendoring uses it. I'll follow up with inlining if the same new function extracted in this PR can't be used. Using Pex code during vendoring is very fragile! As such, that seemed too much to take on in this PR.

Reply:
Reasonable! Although probably worth at least documenting why both functions exist, if they do continue to exist.

Reply:
They won't. It'll be inline or else kill and use new.

Reply:
https://github.com/pantsbuild/pex/pull/1865